ui/src/components/CopyUrlButton.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { CopyToClipboard } from "react-copy-to-clipboard";
+import { PrimaryButton, TerraTooltip } from "libs/common";
+import "@clr/icons/clr-icons.css";
+
+const style = {
+  copyUrlButton: {
+    marginRight: "16px"
```

Comment:
[React will automatically append a "px" suffix](https://reactjs.org/docs/dom-elements.html) so by convention we do this, to be slightly cleaner:
```
marginRight: 16
```